deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```diff
@@ -600,18 +589,8 @@ service:
         - fluentforward
 {{ end }}
 {{ end }}
-## Use this pipeline for cloudwatch logs
```

Comment:
removed the cloudwatch logs pipeline from the metadata config. Cloudwatch logs will now re-use the container logs pipeline.